node/src/controllers/booksController.ts

```diff
@@ -0,0 +1,76 @@
+import 'dotenv/config';
+import { Request, Response } from 'express';
+import { pool } from '../middlewares/db';
+
+export interface BookInfo {
+  book_id : number;
+  title : string;
+  author : string;
+  is_available : boolean;
+}
+
+export const getBook = async (req: Request, res: Response): Promise<void> => {
+  const { available, author } = req.query as { available?: string; author?: string };
+
+  let availableParsed: boolean | undefined;
+  if (available !== undefined) {
+    const lower = available.toLowerCase();
+    if (lower === 'true' || lower === 'false') {
+      availableParsed = lower === 'true';
+    } else {
+      res.status(400).json({ error: "Invalid 'available' query parameter. Expected 'true' or 'false'." });
+      return;
```

Comment:
```suggestion
      return res.status(400).json({ error: "Invalid 'available' query parameter. Expected 'true' or 'false'." });
```
이왕 void 가는 거 conventional하게